package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ramda.github.io",
-  "version": "0.0.0",
+  "version": "0.19.1",
```

Comment:
I thought it seemed like a nice common area to store/access the version number for building the JSDocs.

My other thought was that it may be possible to reduce both the docs and the main library to using a single package.json later.
The doc building requirements could be stored as `devDependencies` so they do not clutter standard installs.